semantics/src/namespace/scopes.rs

```diff
@@ -34,7 +34,7 @@ pub enum ContractDef {
 }
 
 #[derive(Clone, Debug, PartialEq)]
-pub enum FunctionDef {
+pub enum BlockDef {
     Base(Base),
```

Comment:
It would probably be better to have a single `Var` definition variant here, instead of a variant for each variable type. I modified this in my strings PR, so there's no need to update it here. I just wanted to mention this.